source/common/stream_info/utility.cc

```diff
@@ -39,7 +39,7 @@ void ResponseFlagUtils::appendString(std::string& result, const std::string& app
 const std::string ResponseFlagUtils::toShortString(const StreamInfo& stream_info) {
   std::string result;
 
-  static_assert(ResponseFlag::LastFlag == 0x100000, "A flag has been added. Fix this code.");
+  static_assert(ResponseFlag::LastFlag == 0x200000, "A flag has been added. Fix this code.");
```

Comment:
Fix the code per the comment and add tests. Same elsewhere.

Reply:
All of this type of code had already changed from `0x100000` to `0x200000`.

Reply:
Please read the files below and you will see there is more work to do in each of these places. If further comments would have helped you see this better please add them. Thank you!